pkg/crc/oc/clusteroperator.go

```diff
@@ -47,31 +55,31 @@ func GetClusterOperatorStatus(oc OcConfig) (bool, error) {
 			case "Available":
 				if con.Status != "True" {
 					logging.Debug(c.Metadata.Name, " operator not available, Reason: ", con.Reason)
-					allAvailable = false
+					cs.Available = false
```

Comment:
I think I would keep the name `AllAvailable`. The other boolean are `true` when at least one operator has the state, this one is different as it's `true` only if all operators are available.

Reply:
I believe we discussed this, but I forgot what the conclusion was.

Reply:
@cfergeau `ClusterStatus.Available  make more sense to ClusterStatus.AllAvailable since ClusterSTatus already mean the whole cluster state instead of a single operator state.` was the end of the discussion and we both agreed to it.